src/components/footer-system/README.md

```diff
@@ -1,26 +1,254 @@
-# Footer System
-The footer system is made up of three modules used to provide supplemental links and information about your site or app.
+# Footer
 
-## Footer system modules
-1. **Social module** - this module displays icon links to social media profiles
-2. **Resources module** - this module can be divided into columns using the Rivet grid. Columns contain lists of links to resources on your site or external links to outside resources.
-3. **Base module** - This in the only required module of the footer. It contains links to policy information about accessibility, privacy, and copyright.
+Add an IU-branded footer to your website or application.
 
-## States and roles
-The Social and Resources modules must have a `role="complementary"` attribute to create landmarks for assistive technology users. No matter which combination of the Social, Resources, and Base modules you use the first module in the footer, must be a direct sibling of the `<main>` element of the document.
+## Overview
+
+Branding guidelines require each site to have an IU-branded footer. For more information on web branding guidelines, visit [brand.iu.edu](brand.iu.edu).
+
+## Usage
+
+A footer is made up of three parts: the **base footer**, the **resources module**, and the **social module**.
+
+> Only the base footer is required. The resources and social modules are optional.
+
+### Base footer (required)
```

Comment:
I stubbed out a "--light" variant for the footer base module which makes the background (of that module only) white and makes the text black. This may or may not be something we want to look at for all three modules, but for now I made the light variant of the base module as a direct replacement/alternative for the existing stripped-down/application-focused Rivet 1.x footer. See screenshot:

![image](https://user-images.githubusercontent.com/5590034/113619572-25fa8c80-9627-11eb-8cf6-40e6c8553506.png)

We may want to make a note about that here, just so it's on our radar to include in any alpha documentation we push out. You can add the `rvt-footer-base--light` modifier class to see how it works.

![image](https://user-images.githubusercontent.com/5590034/113619753-60642980-9627-11eb-97de-fb7247b36f45.png)

Reply:
Great! I'll add a note about this to the docs.